qa/rpc-tests/feature_fedpeg.py

```diff
@@ -74,21 +60,40 @@ def setup_network(self, split=False):
                     "-addresstype=legacy", # To make sure bitcoind gives back p2pkh no matter version
                     "-deprecatedrpc=validateaddress",
                 ])
-            else:
+            elif self.options.parent_type == 'signet':
+                self.parent_chain = 'signet'
+                rpc_u, rpc_p = rpc_auth_pair(n)
+                self.extra_args.append([
+                    "-printtoconsole=0",
+                    "-signet_blockscript=4104db082f6133b7d9cdb7be60d8384c818ae61233c20485b438e8c1d7772b7f2dbdc04f2f7273d534df0e6a54c8b333994f6275bb921df2438e9a672438e4ed4969ac",
+                    "-signet_siglen=77",
+                    "-signet_seednode=178.128.221.177",
+                    "-port=" + str(p2p_port(n)),
+                    "-rpcuser=" + rpc_u,
+                    "-rpcpassword=" + rpc_p,
+                    "-rpcport=" + str(rpc_port(n)),
+                    "-addresstype=legacy", # To make sure bitcoind gives back p2pkh no matter version
+                    "-deprecatedrpc=validateaddress",
+                    "-fallbackfee=0.00001",
+                ])
+            elif self.options.parent_type == 'elements':
                 self.parent_chain = 'parent'
                 self.extra_args.append([
                     "-printtoconsole=0",
                     '-validatepegin=0',
                     '-anyonecanspendaremine',
                     '-initialfreecoins=2100000000000000',
                 ])
+
             self.binary = self.options.parent_binpath if self.options.parent_binpath != "" else None
             self.nodes.append(start_node(n, self.options.tmpdir, self.extra_args[n], binary=self.binary, chain=self.parent_chain))
+            if self.options.parent_type == 'signet':
+                self.nodes[n].importprivkey('8Hh8jNjkx1aSCgEk3iq9Vo2APZUSDQJVt3rJ2BRpb5Tavqb68vW')
```

Comment:
Is this magic value related to the one in `-signet_blockscript`? Perhaps define them as variables somewhere earlier then together so that it's clear they are related.

Reply:
This will eventually go away, I assume, unless we agree on a global default signet (seems unlikely, at least for upstream to bitcoin/bitcoin).

Reply:
yeah, we could generate the block signing script and the wif dynamically by starting a node, getnewaddress, dumpprivkey, etc beforehand and then starting the actual testing nodes with the resulting script.
This is just the wif for the public key corresponding to the script.

Reply:
This is what I was using for generating the p2pk sript and the corresponding wif to import:

https://github.com/jtimon/elements/commit/3a597eb472868affd94c470568cbedd90a6e42f7